docs/basisterminologie.md

```diff
@@ -0,0 +1,184 @@
+
+Gegevens die in een basisregistratie niet kunnen worden opgenomen, maar (vaak) wel te relateren zijn aan een in een basisregistratie geregistreerde persoon of organisatie noemen we wel ‘aangehaakte’ of ‘plus’gegevens. Partijgegevens zijn hiervan een voorbeeld en gelden als binnen de hele gemeente te gebruiken ‘default’contactgegevens die gebruikt om contact op te nemen met personen of organisaties. Dit betekent dat sprake is van ‘duurzame’ gegevens, die een bestaansrecht hebben dat langer kan duren dan het afhandelen van de vraag of het verzoek waarbij ze (voor het eerst) zijn verstrekt.
+
+Dat sprake is van contactgegevens markeert het verschil tussen qua vorm op elkaar lijkende gegevensverzamelingen zoals ‘namen’ of ‘adressen’ die zowel in basisregistraties als bij partijen voorkomen: hun vastlegging heeft een andere aanleiding en dient een ander doel. De persoonsnaam van een als partij geregistreerde persoon is de naam die deze persoon in contact met de gemeente wil gebruiken. Dit hoeft dus niet dezelfde persoonsnaam te zijn als in de Basisregistratie personen is opgeslagen. Hetzelfde geldt voor adresgegevens. Het bezoekadres van een partij is het adres waar die partij zelf heeft aangegeven bezoek te willen ontvangen. Dit adres hoeft dus niet overeen te komen met een BRP-verblijfsadres of een in het Handelsregister bekend vestigingsadres. In sommige situaties mag echter voor correspondentie of bezoek geen adres worden gebruikt dat afwijkt van bijvoorbeeld het in de BRP bekende verblijfsadres van een persoon. Hiermee moet bij de inrichting van processen rekening worden gehouden.
+
+Van wat we ‘digitale adressen’ noemen bestaan in de basisregistraties slechts beperkt equivalenten. Digitale adressen dienen om bereikbaarheidsgegevens vast te leggen die niet zijn gebonden aan een locatie in de fysieke wereld (waar een bezoekadres, correspondentieadres of postbus wél horen bij zo'n fysieke locatie). Denk hierbij aan e-mailadressen, telefoonnummers of op contactgegevens horend bij gebruikersprofielen op specifieke sociale mediaplatforms.
+
+Dat partijen een ‘relatie’ met de gemeente hebben, betekent dat de gemeente de persoon of organisatie achter een partij moet ‘kennen’. De gemeente weet dus over welke persoon of organisatie partijgegevens gaan, en dat degene die ze heeft verstrekt en eventueel voor hergebruik toestemming heeft gegeven daartoe gerechtigd was. Concreet betekent dit dat partijgegevens alleen geregistreerd mogen worden nadat met voldoende zekerheid is vastgesteld dat:
```

Comment:
Eventueel ?  Volgens mij worden alleen gegevens van een partij vastgelegd als er akkoord is gegeven voor hergebruik. Als dat niet zo is dan worden de betreffende gegevens toch opgenomen als betrokkene bij klantcontact ?

Reply:
Zie de opmerking hierboven. Aanvullend weten we (of in ieder geval ik?) niet of het in alle gevallen nodig is voor hergebruik van contactgegevens expliciet toestemming te verkrijgen.